OutOfSchool/OutOfSchool.WebApi/Controllers/V2/WorkshopController.cs

```diff
@@ -233,28 +221,44 @@ public async Task<IActionResult> GetByFilter([FromQuery] WorkshopFilter filter)
         /// <response code="400">If the model is invalid, some properties are not set etc.</response>
         /// <response code="401">If the user is not authorized.</response>
         /// <response code="403">If the user has no rights to use this method, or sets some properties that are forbidden to change.</response>
+        /// <response code="413">If the request break the limits, set in configs.</response>
         /// <response code="500">If any server error occures.</response>
         [HasPermission(Permissions.WorkshopEdit)]
-        [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(WorkshopDTO))]
+        [ProducesResponseType(StatusCodes.Status200OK)]
```

Comment:
better to leave the type in produces response. If you are using anonymous object then create a type for it.

Reply:
I agree with you that we should use the concrete type, but for this I will create new classes only for returning from controllers (or using in tests)